runtime/executor/src/main/java/edu/snu/nemo/runtime/executor/data/SerializerManager.java

```diff
@@ -51,23 +50,22 @@ public SerializerManager() {
    *
    * @param runtimeEdgeId id of the runtime edge.
    * @param coder         the corresponding coder.
-   * @param propertyMap   ExecutionPropertyMap of runtime edge
+   * @param compressionProperty   compression property
    */
   public void register(final String runtimeEdgeId,
                        final Coder coder,
-                       final ExecutionPropertyMap propertyMap) {
+                       final Optional<CompressionProperty.Value> compressionProperty) {
```

Comment:
It would be good to avoid to use `Optional` as an arguments.
(Please check [here](https://stackoverflow.com/questions/31922866/why-should-java-8s-optional-not-be-used-in-arguments).)
How about separate the method w/ and w/o compression?